curiouslearning/FeedTheMonsterJS
# Fix: Cr 149- Amharic language build

# Changes
- ftm_amharic.json

# How to test
- Check level number 62 in amharic language.

Ref: [CR-149](https://curiouslearning.atlassian.net/jira/software/projects/CR/boards/7?selectedIssue=CR-149)


[CR-149]: https://curiouslearning.atlassian.net/browse/CR-149?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
